website/docs/security/secrets-management/secrets-manager/create-secret.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Storing secrets in AWS Secrets Manager"
+sidebar_position: 61
+---
+
+To store a secret in AWS Secrets Manager, you can either use the AWS Console or the AWS CLI. We will cover both scenarios below.
+
+### AWS Console
```

Comment:
Maintaining console screenshots is more overhead, lets just remove this section and stick to the CLI

Reply:
Agreed!